apps/src/lib/client/rpc.rs

```diff
@@ -338,8 +340,12 @@ pub async fn query_transparent_balance(
                             balance
                         );
                     }
-                    Err(e) => {
-                        display_line!(context.io(), "Querying error: {e}");
+                    Err(namada_sdk::error::Error::Query(
+                        QueryError::NoSuchKey(_),
+                    )) => {
+                        return;
```

Comment:
I think this case should actually print the "No balance found" msg and any other error should still be printed as `edisplay_line!(context.io(), "Querying error: {e}");`